airflow/providers/google/cloud/hooks/cloud_run.py

```diff
@@ -175,4 +175,6 @@ def get_conn(self):
         return self._client
 
     async def get_operation(self, operation_name: str) -> operations_pb2.Operation:
-        return await self.get_conn().get_operation(operations_pb2.GetOperationRequest(name=operation_name))
+        return await self.get_conn().get_operation(
+            operations_pb2.GetOperationRequest(name=operation_name), timeout=120
```

Comment:
Should we make this `120` configurable?

Reply:
@Lee-W Thank you for the comment, I don't think it makes sense to make it configurable. By default the operation is never expected to timeout. The timeout I added was just to make sure that the client has enough time to fetch the operation status from the CloudRun service/database in case of (let's say) load.